src/test/java/net/snowflake/ingest/streaming/internal/datatypes/StringsIT.java

```diff
@@ -0,0 +1,92 @@
+package net.snowflake.ingest.streaming.internal.datatypes;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class StringsIT extends AbstractDataTypeTest {
+  private static final int MAX_STRING_SIZE = 16 * 1024 * 1024;
```

Comment:
Reuse MAX_STRING_LENGTH?

Reply:
I removed this constant, it was not used. But I cannot reuse `BYTES_8_MB` in IT tests because `DataValidationUtil` is package-private and `StringsIT` is in a different package. Anyway, in tests, it is better not to depend on variables from the main codebase.